src/IdentityInterface.php

```diff
@@ -17,6 +17,7 @@
 namespace Authorization;
 
 use ArrayAccess;
+use Authentication\IdentityInterface as AI;
```

Comment:
This makes authorization depend on authentication, which isn't great. Should we add an IdentityInterface to CakePHP core? That would allow both plugins to leverage some common traits and authorization could layer on `can()` and `applyScope`, or does that just move the problem around?

Reply:
In fact, changing `identity` attribute with another object from `Authorization` already creates implicit dependency on `Authentication`. So create separate attribute (like `guard`, or `authorizer`) which implements `Authorization\IdentityInterface` could be the best option if we want to use `Authentication` and `Authorization` togather. Right now this bug real stopper.